haskell-tree-sitter.cabal

```diff
@@ -26,21 +26,14 @@ library
                      , TreeSitter.Node
                      , TreeSitter.Tree
                      , TreeSitter.Ptr
-  Include-dirs:        includes                     
-  Includes:            tree_sitter_ptr.h                     
+  Include-dirs:        includes
+                     , vendor/tree-sitter/lib/include
+                     , vendor/tree-sitter/lib/src
+                     , vendor/tree-sitter/lib/utf8proc
```

Comment:
I couldn't tell how these include directories were specified *before*; maybe `cabal` or the C compiler was just somehow inferring them?

Reply:
Oh, nevermind. We were specifying the full path to the header file in each `foreign import` declaration.